### PR TITLE
Feature.additional setup

### PR DESCRIPTION
**Pull request by bjornborg**

### CODE REVIEW

File: arm-bbb/get.sh [hunk 1]
```diff
@@ -6,4 +6,5 @@ wget ${ROOT_URL}/{install,install-conf,install-chroot,install-post}.sh
 
 mkdir setup-available
 cd setup-available
-wget  ${ROOT_URL}/setup-available/setup-post-01-pru.sh ${ROOT_URL}/setup-available/setup-post-02-docker.sh ${ROOT_URL}/setup-available/setup-user-01-opendlv.sh
+wget  ${ROOT_URL}/setup-available/setup-user-01-opendlv.sh
```
Comment (olbender) [inline arm-bbb/get.sh]: You should instead run the get.sh generation script in the repo root.

File: arm-bbb/setup-available/setup-post-02-adc.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+source install-conf.sh
+
+echo -e "#!/bin/bash\necho -e 'BB-ADC' > /sys/bus/platform/devices/bone_capemgr/slots" > /root/boot/adc.sh
+chmod 755 /root/boot/adc.sh
+echo -e "[Unit]\nDescription=Enables reading values from analogue DC from pins. Values can be read by 'cat /sys/bus/iio/devices/iio\:device0/in_voltage*'.\n\n[Service]\nExecStart=/root/boot/adc.sh\n\n[Install]\nWantedBy=multi-user.target " > /etc/systemd/system/adc.service 
+
+systemctl enable adc.service
```
Comment (olbender) [inline arm-bbb/setup-available/setup-post-02-adc.sh]: Change name of the service to: capemgr-slots-adc.service

File: arm-bbb/setup-available/setup-post-02-i2c.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+source install-conf.sh
+
+echo -e "#!/bin/bash\nchmod 666 /dev/i2c-2" > /root/boot/i2c.sh
+chmod 755 /root/boot/i2c.sh
+echo -e "[Unit]\nDescription=Enables reading and writing to i2c-2 bus to all users. The bus can be diagnosed using i2c-tools.\n\n[Service]\nExecStart=/root/boot/i2c.sh\n\n[Install]\nWantedBy=multi-user.target " > /etc/systemd/system/i2c.service 
+
+systemctl enable i2c.service
```
Comment (olbender) [inline arm-bbb/setup-available/setup-post-02-i2c.sh]: This is fine.

**Review by olbender — CHANGES_REQUESTED** [review]

**Comment by olbender** [comment]
Could you fix this as well (it's related): https://github.com/chalmers-revere/opendlv.os/blob/master/arm-bbb/setup-available/setup-post-01-pru.sh#L91

**Comment by olbender** [comment]
Also read the first comment (re-run the generation script for get.sh)

File: arm-bbb/get.sh (hunk 1 above)
Comment (bjornborg) [reply arm-bbb/get.sh]: Oh alright, then the README should also be updated :)

**Comment by bjornborg** [comment]
The requests should be satisfied.

File: arm-bbb/setup-available/setup-post-01-pru.sh [hunk 4]
```diff
@@ -86,6 +86,3 @@ cd ../../utils/pasm_source
 source linuxbuild
 mv ../pasm /usr/bin
 chmod +x /usr/bin/pasm
-
-# TODO: Make automatic, or do it in C++ instead.
-echo BB-BONE-PRU > /sys/devices/platform/bone_capemgr/slots
```
Comment (olbender) [inline arm-bbb/setup-available/setup-post-01-pru.sh]: I had hoped for a capemgr-slots-pru.service :)

Comment (bjornborg) [reply arm-bbb/setup-available/setup-post-01-pru.sh]: Oh :) I'll fix it.

**Comment by olbender** [comment]
Sorry for being a bit unclear before. I don't think we should add an extra script for the PRU, simply add that service to the end of the existing PRU script. Also revert the name of the ADC script, it's good if it's called: arm-bbb/setup-available/setup-post-02-adc.sh

**Comment by olbender** [comment]
Ah, one more thing. The indices must be unique. Now both the ADC and the I2C scripts have 02 as index.

**Review by olbender — APPROVED** [review]
Great!

**Comment by chrberger** [comment]
Next time you should sit next to each other :-P

**Comment by bjornborg** [comment]
The office next to Ola's is apparently not close enough :).

**Pull request merged**